src/gwt/src/org/rstudio/studio/client/server/remote/RemoteServer.java

```diff
@@ -1390,6 +1391,12 @@ public void createShinyApp(String appName,
       sendRequest(RPC_SCOPE, CREATE_SHINY_APP, params, requestCallback);
    }
    
+   public void getActiveDocumentContextCompleted(JsObject context,
+                                                 VoidServerRequestCallback requestCallback)
```

Comment:
Here you want to declare the interface with ServerRequestCallback<Void> then let the caller decide that they want to use VoidServerRequestCallback to satisfy this interface (it includes automagic error handling behavior that the caller may or may not want to take advantage of)